### DISCUSSION TITLE
Add a link to https://app.simplelogin.io/dashboard/ in Safari extension.

### DISCUSSION THREAD

**Discussion by mkraft** [simple-login/app#1561] [Feature Requests]
Hello, I would like to request a new feature to include a link to https://app.simplelogin.io/dashboard/ within the Safari extension please. Ideally I wouldn't need to login again, but even just the link would be great.

**Comment by nguyenkims** [comment]
@mkraft we are rewriting the safari extension so it can be on par with chrome/firefox extension. Please stay tuned.

**Comment by pbernery** [comment]
Looking forward to see this new extension, especially because I cannot find where is the "API Keys" in the settings.
